tests/libtest.sh

```diff
@@ -29,6 +29,14 @@ else
   test_builddir=$(dirname $0)
 fi
 
+# We need to get the real path in order to use valgrind
+if test -n "${OT_TESTS_UNINSTALLED:-}"; then
```

Comment:
You set this env var in `TESTS_ENVIRONMENT`, so maybe this `if` clause is not required, but instead set the `OSTREE` variable to that libtool output unconditionally? Unless we support overriding the `TESTS_ENVIRONMENT` variable.

Other idea could be creating a directory where we could put a symlink named `ostree` and point it to the `.libs/lt-ostree`, then prepend that directory to PATH env var. This would avoid the necessity of updating all the tests.